lua/lundump.c

```diff
@@ -238,8 +238,8 @@ static void LoadHeader (LoadState* S)
 	"read version %d.%d; expected at least %d.%d",
 	S->name,V(version),V(VERSION0));
  S->swap=(luaU_endianness()!=LoadByte(S));	/* need to swap bytes? */
```

Comment:
A lot of the code in this file should really be replaced by code from xoreos. In particular `luaU_endianness` is horrible. And the slow/evil byte-swapping functions in general. Not necessary for this PR, but it really should be done in the future going forward.